test/system/capability.test.js

```diff
@@ -1,16 +1,16 @@
 const test = require('ava')
 const goshawkdb = require('../..')
-const {setupThenTransactionTest, firstRoot} = require('../helpers/utils')
+const {setupThenTransactionTest, testRootName} = require('../helpers/utils')
 
 test("Capabilities are enforced.",
 	setupThenTransactionTest(
 		(t, connection, txn) => {
-			const root = firstRoot(txn)
+			const root = txn.roots[testRootName]
```

Comment:
(Just commenting on this one as it's the first one I've come across)
Will this error if the root doesn't exist?